### ISSUE TITLE
RFE: mrack man page

### ISSUE THREAD

**Issue by Tiboris** [neoave/mrack#197]
It would be nice to generate or create manpage for mrack.

https://bugzilla.redhat.com/show_bug.cgi?id=2134387

**Comment by dav-pascual** [comment]
Good point. In general, mrack documentation has definetely room for improvement
